pubpub/pubpub
# update: use valid ids and update click-to-copy tooltip for unstable ids

This PR updates the click-to-copy button to disable itself in the case that its target node has an unstable identifier (i.e. the IDs plugin hasn't had a chance to run against the document yet). Upon hovering the disabled button the user will see a tooltip instructing them to create a new release to link to the node.

This PR also adds an alphabetical prefix (`p`) to IDs generated with the ID plugin to conform with the [HTML spec](https://www.w3.org/TR/html401/types.html#type-name).

_Test Plan_

- Open a Pub that was created prior to the ID plugin changes.
- Hover a paragraph element. The click-to-copy button should be disabled.
- Hovering the click-to-copy element should show the text "'You must create a new release to link to this block."
